refinery/tool_manager/test_views.py

```diff
@@ -600,7 +602,7 @@ def test_tool_creation_with_same_display_name_yields_helpful_error(self):
         self.assertEqual(
             "A Tool already exists with a display_name of: '{}'".format(
                 display_name
-            ),
+            ).encode(),
             self.post_response.content
```

Comment:
`self.post_response.content.decode()` instead?

Reply:
I thought it better to leave the `content` untouched.